src/component/menu/Footer/ApiDetails.helpers.tsx

```diff
@@ -0,0 +1,30 @@
+import { IPartialUiConfig } from './ApiDetails';
+
```

Comment:
This looks good. Just a sidenote, I usually group the child components in folders containing files pertaining to that component, and keep the helpers name lowercase (we use uppercase to distinguish component files): 

```
-- Footer
   -- Footer.tsx
   -- ApiDetails
       -- ApiDetails.tsx
       -- apidetails.helpers.ts
```

Reply:
Ah, that makes sense. Should we use `helpers.ts` or `api-details.helpers.ts` in this case?

Reply:
Previously I've used helpers.ts, but it can be a bit tedious if we have a lot of helper.ts files (though I've always been able to locate it by searching for the component name). I don't have strong feelings in this regard, so feel free to do what feels best to you.

Reply:
I kinda like apidetails.helpers.ts